src/Components/FavPage/FavPage.jsx

```diff
@@ -1,11 +1,34 @@
+import { useSelector, useDispatch } from 'react-redux';
+import { FavCitiesList, Message } from '../';
+
 import styles from './FavPage.module.scss';
 
 function FavPage() {
-  return (
-    <main className={styles.container}>
-      <h2>Favourite Locations</h2>
-    </main>
-  );
+  const favCityList = useSelector((state) => state);
+
+  const listItems = favCityList.map((item) => {
+    return <FavCitiesList data={item} key={item.locationData.id} />;
+  });
+
+  function renderComponent() {
+    if (listItems.length !== 0) {
+      return (
+        <main className={styles.container}>
+          <h2 className={styles.title}>Favourite Locations</h2>
+          <div className={styles['container-items']}>{listItems}</div>
+        </main>
+      );
+    } else {
+      return (
+        <Message
+          message1={'Here you can see a list of the weather from your favourite cities.'}
```

Comment:
forgot to add before few suggestions about `Message` component: 
1.`message1` and `message2` prop names are not very clear, what's the difference between them?
2. why not use one `Message` component twice?
<Message>{'Here you can see a list of the weather from your favourite cities.'}</Message>
<Message>{'Your list is empty. Search for a city and add it as a favourite!'}</Message>
3. or why not use `children` prop for content and pass any jsx inside `Message` component?

Reply:
Ok, thank you for suggestions, I've done the changes.